chore: update flutter v2 docs to use plugin options (#7336)
aws-amplify/docs: src/fragments/lib/analytics/flutter/record.mdx

## Flush events

> Events have default configuration to flush out to the network every 30 seconds. To modify the default value, assign the `autoFlushEventsInterval` property within the `options` parameter of the `AmplifyAnalyticsPinpoint()` constructor to your desired duration. This code snippet configures the plugin to flush events every 10 seconds.

Reviewer: *nit:* this is a bit wordy. I think you trim it down since you have the example.

```suggestion
Events have default configuration to flush out to the network every 30 seconds. To modify the default value, assign the `autoFlushEventsInterval` property to your desired duration. The example below configures the plugin to flush events every 10 seconds.
```
